api/api.go

```diff
@@ -83,3 +87,24 @@ func ReadJSONBody(body io.ReadCloser, v interface{}) error {
 
 	return nil
 }
+
+func (api *API) latestAPIRoutes(permissions AuthHandler) {
+	switch api.cfg.LatestVersion {
+	case "v1":
+		api.v1RoutesAsLatest(permissions)
+	default:
+		// Set default in case app configuration is set to non-existant version
+		api.v1RoutesAsLatest(permissions)
```

Comment:
If the user is directed to using the latest version, it is important for the API user to be informed of this. They will know that the version they used is invalid or deprecated then